lib/site_prism/addressable_url_matcher.rb

```diff
@@ -115,28 +119,32 @@ def substitutions
     end
 
     def reverse_substitutions
-      @reverse_substitutions ||= slugs.each_with_index.reduce({}) do |memo, slug_index|
-        slug, index = slug_index
-        memo.merge(slug_prefix(slug) + substitution_value(index) => slug, substitution_value(index) => slug)
-      end
+      @reverse_substitutions ||=
+        slugs.each_with_index.reduce({}) do |memo, slug_index|
+          slug, index = slug_index
+          memo.merge(slug_prefix(slug) + substitution_value(index) => slug,
```

Comment:
if we're expanding the brackets could we go down the
```
method(
  stuff
)
```

route

Reply:
yes, sure we can, but again we have cop to check method length < 11 lines

Reply:
Shouldn't be affected.

Reply:
And alert this one as well